### DISCUSSION TITLE
SDK for Android and iOS

### DISCUSSION THREAD

**Discussion by Jengro777** [tolgee/tolgee-platform#1470] [Ideas]
Now the client is more and more important. Is there any plan to provide Android, iOS and Flutter SDK

**Comment by JanCizmar** [comment]
Yes, it's in the roadmap for the next year. 🪴  https://tolgee.io/roadmap

**Reply by Jengro777** [reply]
Great, and the update frequency of tolgee is much faster than that of traduora. I will recommend tolgee to my team

**Reply by JanCizmar** [reply]
Glad to hear!
